README.md

```diff
@@ -12,36 +12,27 @@ Runner](https://github.com/marten-seemann/quic-interop-runner).
 
 ## Quickstart
 
-You will need to have golang installed.
+You will need to have Go 1.15+ installed.
 
-You will need to clone this repository on the `src` directory of your
-`$GOPATH`. To learn your `$GOPATH`, use `go env`.
-
-Tests require certificates and other cryptographic artifacts to be generated
-beforehand.
+0. Clone this repository to the `src` directory of your `$GOPATH`.
```

Comment:
By using the magic of Markdown, you don't need to list the '0', '1', etc, but rather just use '1': https://www.markdownguide.org/basic-syntax/#ordered-lists See the second example.

Reply:
It would be less error-prone to just use '1', but my concern was that it might be slightly confusing for someone reading the file in plaintext.